src/output_handlers/s3.py

```
@@ -0,0 +1,76 @@
+        Asynchronously writes a single entry to the bucket.
+        """
+        data = json.dumps(entry).encode('utf-8')
+        try:
+            response = await self.client.put_object(
+                Bucket = self.bucket_name,
+                Key = f"{uid}.json",
+                Body = data
+            )
+
+            if response['ResponseMetadata']['HTTPStatusCode'] not in (200, 201):
+                raise Exception(f"Response: {response}")
```

Ruff (0.14.0)

Create your own exception

(TRY002)